website/pages/en/quick-start.mdx

```diff
@@ -35,133 +30,141 @@ Using [yarn](https://yarnpkg.com/):
+If you’d like to examine your subgraph before publishing it to the network, you can use [Subgraph Studio](https://thegraph.com/studio/) to do the following:
+
+- Run a sample query.
+- Analyze your subgraph in the dashboard to check information.
+- Check the logs on the dashboard to see if there are any errors with your subgraph.
+- The logs of an operational subgraph will look like this:
 
 ![Subgraph logs](/img/subgraph-logs-image.png)
 
-If your subgraph is failing, you can query the subgraph health by using the GraphiQL Playground. Note that you can leverage the query below and input your deployment ID for your subgraph. In this case, `Qm...` is the deployment ID (which can be located on the Subgraph page under **Details**). The query below will tell you when a subgraph fails, so you can debug accordingly:
-
-```graphql
-{
-  indexingStatuses(subgraphs: ["Qm..."]) {
-    node
-    synced
-    health
-    fatalError {
-      message
-      block {
-        number
-        hash
-      }
-      handler
-    }
-    nonFatalErrors {
-      message
-      block {
-        number
-        hash
-      }
-      handler
-    }
-    chains {
-      network
-      chainHeadBlock {
-        number
-      }
-      earliestBlock {
-        number
-      }
-      latestBlock {
-        number
-      }
-      lastHealthyBlock {
-        number
-      }
-    }
-    entityCount
-  }
-}
-```
+### 7. Publish Your Subgraph to The Graph
 
-## 7. Publish your subgraph to The Graph’s Decentralized Network
+Publishing a subgraph to the decentralized network makes it available for [Curators](/network/curating/) to begin curating it and [Indexers](/network/indexing/) to index it.
 
-Once your subgraph has been deployed to Subgraph Studio, you have tested it out, and you are ready to put it into production, you can then publish it to the decentralized network.
+#### Publishing with Subgraph Studio
 
-In Subgraph Studio, you will be able to click the publish button on the top right of your subgraph's page.
+1. To publish your subgraph, click the publish button in the dashboard.
+2. Select the network to which you would like to publish your subgraph.
+   - Subgraphs are published to Arbitrum One to take advantage of the [faster transaction speeds and lower gas costs](https://thegraph.com/docs/en/arbitrum/arbitrum-faq/).
 
-Select the network you would like to publish your subgraph to. It is recommended to publish subgraphs to Arbitrum One to take advantage of the [faster transaction speeds and lower gas costs](/arbitrum/arbitrum-faq).
+#### Publishing from the CLI
 
-The (upgrade Indexer)[/sunrise/#about-the-upgrade-indexer] will begin serving queries on your subgraph regardless of subgraph curation, and it will provide you with 100,000 free queries per month.
+As of version 0.73.0, you can also publish your subgraph with the [`graph-cli`](https://github.com/graphprotocol/graph-tooling/tree/main/packages/cli).
```

Comment:
Why do we link to the Graph CLI in GitHub now? It was mentioned earlier in the page. I would just remove it:
```suggestion
As of version 0.73.0, you can also publish your subgraph with the Graph CLI.
```